backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala

```diff
@@ -37,7 +37,7 @@ class VeloxBackend extends Backend {
   override def iteratorApi(): IteratorApi = new IteratorHandler
   override def sparkPlanExecApi(): SparkPlanExecApi = new SparkPlanExecHandler
   override def transformerApi(): TransformerApi = new TransformerHandler
-  override def validatorApi(): ValidatorApi = new Validator
+  override def validatorApi(): ValidatorApi = new VeloxValidator
```

Comment:
Do we have a consistent naming convention for the components? Seems to me that we mixed the use of `...Handler`, `...`, `...Initializer`, `Velox...`

Reply:
Binwei once asked us to remove "velox" prefix inside velox module (maybe, only few exceptions not), for brevity consideration. cc @FelixYBW

Reply:
This change is due to we have a file `VeloxValidator` but with a different class name `Validator`. This pr changes the class name to `VeloxValidator`. I'm fine to change the file name to `Validator` if you have some concern.

Reply:
The latest change is OK to me. Thanks!